packages/playground/website/src/components/playground-viewport/index.tsx

```diff
@@ -49,6 +50,10 @@ export default function PlaygroundViewport({
 		}
 	}, [blueprint?.steps]);
 
+	useEffect(() => {
+		collectWindowErrors(logger);
```

Comment:
Logger is technically a singleton, but this effect may run twice if the PlaygroundViewport is ever mounted twice. Since `collectWindowError` isn't dependent on `playground`, perhaps a top-level call in `website/main.tsx` would be more appropriate?

Reply:
[988a9a5](https://github.com/WordPress/wordpress-playground/pull/1035/commits/988a9a5aaececcd19fde79e88ccb72820fa9813b)